GliaWidgets/SwiftUI/Components/SnackBarView/SerialQueue.swift

```diff
@@ -0,0 +1,56 @@
+import Foundation
+
+/// Represents queue with serial strategy. Not thread-safe.
+final class SerialQueue {
```

Comment:
Seems like this is perfect candidate for unit tests.

Reply:
added tests - f529ff1c062e6c8cf12a429d21cb10dc1f01d838